internal/agent/sender/sender.go

```diff
@@ -5,30 +5,25 @@ import (
 	"github.com/DimaKoz/practicummetrics/internal/common/model"
 	"github.com/go-resty/resty/v2"
 	"log"
-	"net/url"
 )
 
 // ParcelsSend sends metrics
 func ParcelsSend(cfg *config.AgentConfig, metrics []model.MetricUnit) {
 	client := resty.New()
+	targetURL := "http://" + cfg.Address + "/update/"
```

Comment:
Please use efficient formatting of strings

Reply:
done